apps/documentation/src/routes/documentation/routing/api-routes.mdx

```diff
@@ -0,0 +1,173 @@
+import MetaTags from '@/components/MetaTags'
+
+<MetaTags
+  title="Tuono - API routes"
+  canonical="https://tuono.dev/documentation/routing/api-routes"
+/>
+
+import Breadcrumbs from '@/components/Breadcrumbs'
+
+<Breadcrumbs breadcrumbs={[{ label: 'API Routes' }]} />
+
+# API Routes
+
+Just like with [NextJS API routes ](https://nextjs.org/docs/pages/building-your-application/routing/api-routes). Tuono API routes provide a solution to build a public API.
+
+Any file inside the folder `src/routes/api` is mapped to the base url as `/api/*` and will be treated as an API endpoint instead of a page. They are server-side only bundles and won't increase your client-side bundle size.
+
+## HTTP Methods
+
+Tuono provides the following macros to define HTTP Request Methods
+
+```rs
+#[tuono_lib::api(GET)]
+#[tuono_lib::api(POST)]
+#[tuono_lib::api(PUT)]
+#[tuono_lib::api(PATCH)]
+#[tuono_lib::api(DELETE)]
+```
```

Comment:
We don't need this section. Users can easily understand that only the HTTP methods can be passed to it with the example

```suggestion
```